js/popups/tablemenupopup.ts

```diff
@@ -51,10 +51,17 @@ export class TableMenuPopup {
      */
     public ratingSupported = ko.observable(appConfig.game.hasRating);
 
+    /**
+     * Provides information about authentication status for application user.
+     */
+    private authInformation: IAuthenticationInformation;
+
     constructor(
         private currentTableProvider: ICurrentTableProvider,
         private commandExecutor: ICommandExecutor,
-        private accountManager: IAccountManager) {
+        private accountManager: IAccountManager,
+        private authInfo: IAuthenticationInformation) {
```

Comment:
This declaration automatically create field `authInfo` in the class, so no need to declare additional field `authInformation`